tox.ini

```diff
@@ -0,0 +1,61 @@
+[tox]
+envlist =
+    py{36,37,38,39}-test{,-alldeps,-oldestdeps}{,-devastropy}{,-cov}
+    codestyle
+    build_docs
+requires =
+    setuptools >= 30.3.0
+    pip >= 19.3.1
+    tox-pypi-filter >= 0.12
+isolated_build = true
+
+
+[testenv]
+
+# The following option combined with the use of the tox-pypi-filter above allows
+# project-wide pinning of dependencies, e.g. if new versions of pytest do not
+# work correctly with pytest-astropy plugins. Most of the time the pinnings file
+# should be empty.
+pypi_filter = https://raw.githubusercontent.com/astropy/ci-helpers/master/pip_pinnings.txt
+
```

Comment:
This might fix the `_find_home` error. It did get modified twice recently but none of those got backported to 3.x.

```suggestion

# Pass through the following environemnt variables which are needed for the CI
passenv = HOME WINDIR CI

```

If this does not work, I would recommend bumping minversion of astropy to current LTS.